pinot-core/src/main/java/com/linkedin/pinot/core/indexsegment/generator/ColumnPartitionConfig.java

```diff
@@ -48,6 +49,11 @@ public ColumnPartitionConfig(@Nonnull @JsonProperty("functionName") String funct
     _partitionValues = partitionValues;
   }
 
+  public ColumnPartitionConfig(Map<String, String> map, String column, int partitionId, int nPartitions) {
+    _functionName = map.get(column) + PARTITIONER_DELIMITER + nPartitions;
+    _partitionValues = "[" + partitionId + " " + partitionId + "]";
```

Comment:
isn't it safer to call the rangesToString() method that is defined in this class?